Source/SwiftLintFramework/Rules/LeadingWhitespaceRule.swift

```diff
@@ -9,7 +9,23 @@
 import Foundation
 import SourceKittenFramework
 
-public struct LeadingWhitespaceRule: ConfigurationProviderRule, SourceKitFreeRule {
+extension String {
+    private func countOfLeadingCharactersInSet(characterSet: NSCharacterSet) -> Int {
+        var count = 0
+        for char in utf16 {
+            if !characterSet.characterIsMember(char) {
+                break
+            }
+            count += 1
+        }
+        return count
+    }
+    private func leadingNewlineAndWhiteSpaceCount() -> Int? {
```

Comment:
if we add this (which I'm not sure we need to), we should use it where we're currently using the expanded form (e.g. `validateFile(_:)`).